### PR TITLE
[Doc] Unnest UC descriptions

### PR DESCRIPTION
**Pull request by McSinyx**
Fix GH-188, I hope, while stabbing at GH-193 as well.

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
I think you should delete the View result as well? There is no part where we upload "result".

File: docs/source/req/use-case/academic.rst [hunk 1]
```diff
@@ -25,125 +25,34 @@ In addition, academic assistant could also track student progress.
...
-Brief Description
-^^^^^^^^^^^^^^^^^
-
 This use case allows Supervisor to evaluate the project.
 
-Flow of Events
-^^^^^^^^^^^^^^
-
-Basic Flow
-""""""""""
```
Comment (9a24f0) [inline docs/source/req/use-case/academic.rst:37]: I suggest using
```
Brief Description
"""""""""""""""""
```
Idk if the intent is enough for seperating these 2 parts

Comment (Huy-Ngo) [reply docs/source/req/use-case/academic.rst:37]: He intended to remove all of the specs.

Comment (9a24f0) [reply docs/source/req/use-case/academic.rst:37]: Yeah but I think that we should keep the two headers

Comment (McSinyx) [reply docs/source/req/use-case/academic.rst:37]: I suppose Imma solve this by adding a line to connect the two parts.

**Review by Huy-Ngo — APPROVED** [review]
The judges should be able to evaluate the reports so this UCM is outdated, but I think it's not within the scope of this project.

**Comment by McSinyx** [comment]
> it's not within the scope of this ~project~ PR.

FTFY.  Thanks BTW.

**Comment by Huy-Ngo** [comment]
I blame the ~~nonexistent~~ autocorrector

**Pull request merged**